internal/commands/untrust_builder.go

```diff
@@ -0,0 +1,56 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "untrust-builder <builder-name>",
+		Short: "Stop trusting builder",
+		Long:  "Stop trusting builder.\n\nWhen building with this builder, all lifecycle phases will be no longer be run in a single container using the builder image.",
+		Args:  cobra.MaximumNArgs(1),
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 || args[0] == "" {
+				logger.Infof("Usage:\n\t%s\n", cmd.UseLine())
+				return nil
+			}
```

Comment:
This validation should be something that cobra can take care of for us if we were to set the `Args` params with proper requirements.